.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,13 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+XUNIT_PATH="$PWD/sponge_log.xml"
+XUNIT_TMP_PATH="$PWD/drift_tmp.xml"
+
+cp $XUNIT_PATH $XUNIT_TMP_PATH
```

Comment:
Maybe you want to guard it with checking if the file exists.

Reply:
Is there a situation where `pytest` could fail to output an XUnit file?

Reply:
Never mind, added a guard here (for "defensive design" if nothing else).